dev/com.ibm.ws.messaging.comms.client/src/com/ibm/ws/sib/jfapchannel/impl/CommsOutboundChain.java

```diff
@@ -91,12 +116,19 @@ public CommsOutboundChain(
         this.commsClientService = commsClientService;
 
         isSecureChain = MetatypeUtils.parseBoolean(OUTBOUND_CHAIN_CONFIG_ALIAS, "useSSL", properties.get("useSSL"), false);
+        
+        useNettyTransport = ProductInfo.getBetaEdition() ? 
```

Comment:
A trigraph operator does not make this clearer. Use 
`Product.getBetaEdition() && MetatypeUtils.parseBoolean(OUTBOUND_CHAIN_CONFIG_ALIAS, "useNettyTransport", properties.get("useNettyTransport"), true)` instead.

Reply:
Sounds good, I'll make this change.